code/obj/item.dm

```diff
@@ -30,10 +30,12 @@
 	var/c_flags = null
 	var/tooltip_flags = null
 	var/item_function_flags = null
+	var/force_use_as_tool = 0
 
 	pressure_resistance = 50
 	var/obj/item/master = null
 	var/amount = 1
+	var/uses_left
```

Comment:
Only the dye bottle seems to make use of this. Is defining at item level necessary?

Reply:
Forgot about that part. Fixed it!